### PR TITLE
Adding healthcheck end point.

### PR DESCRIPTION
**Pull request by ch-mcabrera**
This is the first commit to add: 
- the files to build and start the service: `Makefile` and `start.sh`
- the `healthcheck` end point.  

`data-document-rendering-api` service has been used as a template to create this new service; since we have gone down the route of using java servlet. 

Resolves: [FAPI-2217](https://companieshouse.atlassian.net/browse/FAPI-2217)

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    echo "Downloading environment from: ${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}"
+    wget -O "${APP_DIR}/private_env" "${CONFIG_URL}/${ENVIRONMENT}/private_env"
+    wget -O "${APP_DIR}/global_env" "${CONFIG_URL}/${ENVIRONMENT}/global_env"
+    wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
+    source "${APP_DIR}/private_env"
+    source "${APP_DIR}/global_env"
+    source "${APP_DIR}/app_env"
+
+    export CHIPS_FILING_API_PORT="${PORT}"
```
Comment (cburns93) [inline start.sh]: We should not be exporting the `PORT` here. We should be passing it through to the App. Look at the way that the data-document-rendering-api has done it within its start.sh script and App.java

Comment (ch-mcabrera) [reply start.sh]: The `data-document-rendering-api` also exports the port.

Comment (cburns93) [reply start.sh]: Not anymore, we did a change to fix that as there were a few problems that arose. https://github.com/companieshouse/data-document-rendering-api/blob/develop/start.sh

Comment (cburns93) [reply start.sh]: For reference: [#PR36](https://github.com/companieshouse/data-document-rendering-api/pull/36)

File: src/main/java/uk/gov/companieshouse/chipsfilingapi/App.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+public class App {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger("chips-filing-api");
+
+    public static void main(String[] args) {
+
+        Integer portNumber = getChipsFilingApiPort();
```
Comment (cburns93) [inline src/main/java/uk/gov/companieshouse/chipsfilingapi/App.java]: You do not need this method anymore.. You can just copy the `data-document-rendering-api` `App.java` and do it all within the main method.

`// Returns the configured ${PORT} value in the `start.sh` script - passed in under `server.port`.
  // If no port is configured, return null
 Integer port = Integer.getInteger("server.port");
if (port == null) {
            LOGGER.info("Failed to start service, no port has been configured");
            System.exit(0);
        }`

Comment (ch-mcabrera) [reply src/main/java/uk/gov/companieshouse/chipsfilingapi/App.java]: The `getChipsFilingApiPort` method contains that logic.

**Pull request merged**